src/PostGraphileManyCreatePlugin.ts

```diff
@@ -142,21 +142,22 @@ const PostGraphileManyCreatePlugin: T.Plugin = (
       const newPayloadHookSpec = {
         name: `mn${inflection.createPayloadType(table)}`,
         description: `The output of our many create \`${tableTypeName}\` mutation.`,
-        fields: ({ fieldWithHooks }) => {
+        fields: ({ fieldWithHooks }: {fieldWithHooks: any}) => {
           const tableName = inflection.tableFieldName(table);
+          const tableNamePlural = `${tableName}s`
```

Comment:
@benjie This might be of interest? It seems Postgraphile does a lot of "pluralizing", is there a function I can use besides my naive strategy of appending an `s`? Thanks! 

Also if you look below, does `type: new GraphQLList(tableType)` make sense?